app.py

```diff
@@ -234,7 +237,8 @@ def search_snap():
 
     return flask.render_template(
         'search.html',
-        **context
+        **context,
+        now=datetime.datetime.utcnow()
```

Comment:
@nottrobin @tbille Is it possible in flask to make sure something (like `now` here) is always added to context by default, without adding it separately in every view?

Reply:
If we were only using `context` in every function we could create a function that creates this object, and that adds each time this var.
Otherwise use JS :P

Reply:
OK, found it myself. It's called `@app.context_processor` ;)

Reply:
@bartaz what do you need the time for? Is this to calculate the year for the copyright notice?

Could I suggest we don't make that dynamic? The copyright *starts* from the year mentioned, it doesn't need to be the latest year:

> The general rule is that the year to include in a copyright notice is the year of first publication of the work. First publication is when the work is made available to the public without restriction.

(From http://www.copyrightlaws.com/copyright-notice-year/)

I think if we're going to use a year in the copyright notice (and I'd love to talk about removing the year, but feat that discussion will delay this unnecessarily), I think the most valid year to use is the current year at the point that the copyright notice is written - *not* the current year. So make i t 2017 now, and it should stay 2017 until we make a new website, or actively decide to update it for another reason.

Reply:
See also: https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

> The date on the notice establishes how far back the claim is made. This means if you update the date, you are no longer claiming the copyright for the original date